### DISCUSSION TITLE
Feature Request: Update 'New Entry' dropdown to display Section Group Headings and list Sections in same order as sources sidebar

### DISCUSSION THREAD

**Discussion by BenParizek** [craftcms/cms#8566] [Ideas]
Craft allows us to group Sections into what we think are logical groups for a project and give groups of sections custom headers:
http://share.barrelstrength.co/3H3z3s091i3t

However, the **New Entry** dropdown always displays all Sections in alphabetical order:
http://share.barrelstrength.co/262f0G0c0e2v

It would be nice to have the grouping we choose in the sources sidebar reflected in the New Entry dropdown. One way to do this could be similar to how Sites are grouped by Site Group in the Site dropdown:
http://share.barrelstrength.co/31243s1m1K0S

**Comment by brandonkelly** [comment]
Technically the sources could include non-sections as well (currently only possible via plugins but eventualy something admins can define in the CP as well), so Craft doesn’t think of them as “section groups”. That said, I guess I’ll leave this open to see if it gains any traction. We could maybe introduce a new “Section Groups” concept, or at least give admins the ability to set custom section ordering.
